%Created on Overleaf
% NL Resume
%----------------------------

\documentclass[11.5pt,a4paper,roman]{moderncv}
\moderncvstyle{banking}  
\moderncvcolor{blue}  
\nopagenumbers{}    

% character encoding
\usepackage[utf8]{inputenc}
\usepackage{fontawesome}
\usepackage{tabularx}
\usepackage{ragged2e}


% adjust the page margins
\usepackage[scale=0.8]{geometry}
\usepackage{multicol}
\usepackage{import}

% personal data
\name{NHI X}{LUU}
\address{\small Silver Spring, MD 20906}{}{}

\vspace*{5mm}  
\newcommand*{\customcventry}[7][.25em]{
  \begin{tabular}{@{}l} 
    {\bfseries #4}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\bfseries #5}
  \end{tabular} \\
  \begin{tabular}{@{}l} 
    {\itshape #3}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\itshape #2}
  \end{tabular}
  \ifx&#7&%
  \else{\\%
    \begin{minipage}{\maincolumnwidth}%
      \small#7%
    \end{minipage}}\fi%
  \par\addvspace{#1}}
  
\newcommand*{\researchentry}[7][.25em]{
  \begin{tabular}{@{}l} 
    {\bfseries #4}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\bfseries #5}
  \end{tabular} \\
  \begin{tabular}{@{}l} 
    {\upshape #3}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\itshape #2}
  \end{tabular}
  \ifx&#7&%
  \else{\\%
    \begin{minipage}{\maincolumnwidth}%
      \small#7%
    \end{minipage}}\fi%
  \par\addvspace{#1}}

\newcommand*{\cvref}[3][.25em]{
%   \vfill\noindent
  \begin{tabular}{@{}l} 
    {\bfseries #2}
  \end{tabular}
  \ifx&#3
  \else{\\%
    \begin{minipage}{\maincolumnwidth}%
      \small#3%
    \end{minipage}}\fi%
  \par\addvspace{#1}}
  

\setlength{\tabcolsep}{12pt}

%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%-----       resume       ---------------------------------------------------------
\vspace*{-19mm}
\makecvtitle
\vspace*{-15mm}
\hspace{1cm}
\begin{center}
\begin{tabular}{ c c c c c }
 \faGlobe\enspace\href{https://www.nhiluu.me/}{About Me} & 
 \faLinkedinSquare\href{https://www.linkedin.com/in/nluu/}{ Nhi Luu} &
 \faEnvelopeO\enspace nluu1@umbc.edu & \faGithub\enspace\href{https://github.com/nluu1}{nluu1} &
 \faMobile\enspace (240) 584-1439\\  
\end{tabular}
\end{center}
\vspace*{-3mm}
\section{EDUCATION}
{\customcventry{Expected: Fall 2023}{B.S., Applied Biotech (TLST) - Bioinformatics; GPA: 4.0}{UMBC at the Universities at Shady Grove}{Rockville, MD}{}{B.S./M.P.S. in Data Science: DATA601}
}

{\customcventry{Jan 2018 - May 2019}{Microbiology - Credits: 32}{University at Maryland - College Park}{College Park, MD}{}{}}

{\customcventry{Dec 2017}{A.S., Life Sciences; GPA: 3.83}{Montgomery College}{Rockville, MD}{}{}
}

\vspace*{-3mm}
\section{TECHNICAL SKILLS \& KNOWLEDGE AREAS}
{\begin{itemize}
\item \textbf{Software}: ClustalX2, BLAST/NCBI, iTOL/Dendroscope, Jalview, ImageJ, MS Office/Teams
\item \textbf{Programming Language}: Python, R, Linux/Command-line, Latex
\item \textbf{Laboratory}: Immunofluorescence, RT-PCR, transformation, cells and spheroids culture, protein purification methods, microbiology techniques, gel electrophoresis, other staining techniques ( H&E, ORO, gram stain)
\item \textbf{Instrumentation}: Spectrophotometer, Nanodrop, Fluorescent/Confocal microscopy, pipetting

  \end{itemize}
}

\vspace*{-3mm}
\section{PROGRAMMING EXPERIENCE}
{\researchentry{08/2022- Present}{\textbf{ExtremeBiome Research Projects} - Bioinformatics Pipeline}{Universities at Shady Grove}{Rockville, MD}{}
{\begin{itemize}    
  \item Develop genomics tools and automate variant calling workflow on Illumina/PacBio NGS data \textit{(OmicsVMconfigure: https://doi.org/10.5281/zenodo.7641805 )}
  \item Publish genomics visualization (circos, statistical charts) on interactive web-apps utilizing R/R-Shiny through virtual machines (VMs)
 \end{itemize}
}
}
\vspace{-\baselineskip}
{\researchentry{04/2022- Present}{\textbf{Student Projects} - Bioinformatics/Data Analysis}{}{}{}
{\begin{itemize}
    \item Construct phylogenetic trees and analyze bacterial taxonomy based on different molecular markers using NCBI/BLAST and alignment tools (ClustalX2, Jalview, iTOL)
    \item Conduct descriptive statistics on Cardiology dataset using R/Python on Blood-work analysis (GitHub)
    \item Lead group project on data analysis and visualization of Maryland Census data in Python using API and web scraping
\end{itemize}
}
}
\vspace*{-3mm}
 \section{RESEARCH EXPERIENCE}
{\researchentry{08/2022- Present}{\textbf{Phage Hunting Intern}}{Adaptive Phage Therapeutics, Inc.}{Gaithersburg, MD}{}
{\begin{itemize}
  \item Utilize aseptic microbiology techniques to enrich, isolate, and purify hunted phages against multi-drug resistant bacterial strains
  \item Communicate resources and findings among colleagues in team and agile independent projects
  \item Perform and adhere to cGMP, GDP in BSL-2 laboratory, safely and efficiently utilize lab procedures, instruments and tools
  \item Participated in planning and supporting training for new phage hunting interns
\end{itemize}
}}
\addvspace{0.1cm}
{\researchentry{07/2021- 10/2022}{\textbf{Research Assistant} – Wound Healing Model Research}{Montgomery College – Biology Department}{Rockville, MD}{}
{\begin{itemize}
  \item Establish optimal conditions for spheroid co-cultures in 3D to observe their interactions and patterns
  \item Perform tissue culture, scratch wound assay, immunofluorescence, H&E staining on different cell lines for image and statistical analysis
  \item Developed and optimized lab protocols for future research projects on spheroids and viability with drugs
\end{itemize}
}}
\vspace{-\baselineskip}
\addvspace{0.1cm}
{\researchentry{01/2017- 12/2017}{\textbf{Student Assistant} – Novel Solutions to Wound Healing Project}{}{}{}
{\begin{itemize}
  \item Independently presented a 20-page Literature Review Presentation on the \textit{ Development of a Full-Thickness Human Skin Equivalent Derived from TERT-Immortalized Keratinocytes and Fibroblasts}
  \item Pioneered ideas to a published literature review on Springer: \textit {Skin wound Healing: Refractory Wounds and Novel Solutions} (first online 24 May 2018)
\end{itemize}
}
}

\setlength{\columnwidth}{\textwidth}
\addtolength{\columnwidth}{-\columnsep}
\setlength{\columnwidth}{.5\columnwidth}

\setlength{\hintscolumnwidth}{0.175\columnwidth}
\setlength{\separatorcolumnwidth}{0.025\columnwidth}
%
\renewcommand*{\recomputecvbodylengths}{%
  % body lengths
  \setlength{\maincolumnwidth}{\columnwidth-\leftskip-\rightskip-\separatorcolumnwidth-\hintscolumnwidth}%
  \setlength{\listitemcolumnwidth}{\maincolumnwidth-\listitemsymbolwidth}%
  \setlength{\doubleitemcolumnwidth}{\maincolumnwidth-\hintscolumnwidth-\separatorcolumnwidth-\separatorcolumnwidth}%
  \setlength{\doubleitemcolumnwidth}{0.5\doubleitemcolumnwidth}%
  \setlength{\listdoubleitemcolumnwidth}{\maincolumnwidth-\listitemsymbolwidth-\separatorcolumnwidth-\listitemsymbolwidth}%
  \setlength{\listdoubleitemcolumnwidth}{0.5\listdoubleitemcolumnwidth}%
  % regular lengths
  \setlength{\parskip}{0pt}}
\recomputecvbodylengths


% \nocite{*}
% \bibliographystyle{plain}
% \bibliography{publications}                        
%-----       letter       ---------------------------------------------------------

\end{document}
